nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -98,11 +99,10 @@ public void handle(String target, HttpServletRequest request, HttpServletRespons
                 File pkgZip = new File(testStore, pkgId + ".zip");
                 if (pkgZip.exists()) {
                     response.setContentLength((int) pkgZip.length());
+                    response.setStatus(HttpServletResponse.SC_OK);
                     try (ServletOutputStream os = response.getOutputStream()) {
-                        byte[] byteArray = FileUtils.readFileToByteArray(pkgZip);
-                        os.write(byteArray, 0, byteArray.length);
+                        IOUtils.copy(new FileInputStream(pkgZip), os);
```

Comment:
The `FileInputStream` is not closed. You must do:

    try (InputStream is = new FileInputStream(pkgZip)) {
        IOUtils.copy(is, os);    
    }